YACS-RCOS/yacs.n
# 805 feature request ability to export schedule to google calendar

**Issue**
Added Login and Sign Up Google Functionality to YACS

*Example:*
> closes #113



*Example:*
> Added test `tests/api/db/test_classinfo.py:test_crn_uniqueness` to ensure all classes returned from API call don't have repeat CRNs

**Test Procedure**

To test this you can login with your google account and it will automatically make an account for you. You can sign out and login with google to access the account you just made.



More info that would help people review your change

The way the password is made is kind of scuffed. People from the backend team should fix it.
